quantum/keymap_introspection.h

```diff
@@ -88,3 +88,22 @@ tap_dance_action_t* tap_dance_get_raw(uint16_t tap_dance_idx);
 tap_dance_action_t* tap_dance_get(uint16_t tap_dance_idx);
 
 #endif // defined(TAP_DANCE_ENABLE)
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Key Interrupt
+
+#if defined(KEY_INTERRUPT_ENABLE)
+#    include "process_key_interrupt.h"
+// Get the number of key interrupts defined in the user's keymap, stored in firmware rather than any other persistent storage
+uint16_t key_interrupt_count_raw(void);
+// Get the number of key interrupts defined in the user's keymap, potentially stored dynamically
+uint16_t key_interrupt_count(void);
+// Get the keycode for the key press, stored in firmware rather than any other persistent storage
+uint16_t key_interrupt_get_keycode_press_at_idx_raw(uint16_t idx);
+// Get the keycode for the key unpress, stored in firmware, potentially stored dynamically
+uint16_t key_interrupt_get_keycode_press_at_idx(uint16_t idx);
+// Get the keycode for the key unpress, stored in firmware rather than any other persistent storage
+uint16_t key_interrupt_get_keycode_unpress_at_idx_raw(uint16_t idx);
+// Get the keycode for the key unpress, stored in firmware, potentially stored dynamically
+uint16_t key_interrupt_get_keycode_unpress_at_idx(uint16_t idx);
```

Comment:
```suggestion
// Get the keycodes for the key interrupt, stored in firmware rather than any other persistent storage
key_interrupt_t key_interrupt_get_raw(uint16_t idx);
// Get the keycodes for the key interrupt, stored in firmware, potentially stored dynamically
key_interrupt_t key_interrupt_get(uint16_t idx);
```